src/components/localization/language-dropdown.tsx

```diff
@@ -66,7 +62,7 @@ const Absolute = styled.div<AbsoluteProps>`
             return '5.5rem'
         }
     }};
-    left: -22rem;
+    left: -7rem;
```

Comment:
I think the border-radius need to be 8px according to the design can we adjust that in line 70?🙏 

<img width="240" alt="Screenshot 2022-09-06 at 7 22 23 PM" src="https://user-images.githubusercontent.com/64970259/188623633-23b9f7fc-57a9-4dfa-ad3e-3a05116ab965.png">

Reply:
Ok, which values are acceptable to be copied from figma? Because sometimes they are not to be copied exactly as they are

Reply:
Thats a case for `rem` I assume for `px` should be fine 🤔